packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -859,9 +1019,9 @@ export function getNotSeenValidatorsFn(state: CachedBeaconStateAllForks): GetNot
       const committee = state.epochCtx.getBeaconCommittee(slot, committeeIndex);
 
       const notSeenCommitteeMembers = new Set<number>();
-      for (const [i, validatorIndex] of committee.entries()) {
+      for (const [committeeValidatorIndex, validatorIndex] of committee.entries()) {
```

Comment:
shouldn't this be called `validatorCommitteeIndex`, ie. the position of the validator in the committee

Reply:
just checked `unstable`, we both have `committeeValidatorIndex` and `validatorCommitteeIndex`
do you know if it's specified somewhere in the spec?